### PR TITLE
[#1] [Chore] Init project using flutter-template 

### LINKED ISSUE #1: [Chore] Init project using flutter-template
## Why

The repository needs to have the initial project setup using flutter-template

## Acceptance Criteria

The repository will have an initial setup and we can run the app

Generate a new project using the flutter template
- Package name should be `co.nimblehq.davidbruce.survey`
- App name should be Survey Flutter

### PR DESCRIPTION
**Pull request by ducbm051291**
- Close #1 

## What happened

- Clone project from [Flutter Template](https://github.com/nimblehq/flutter_templates) 
- Run `make.sh` with the app name and package name
 
## Insight

- N/A

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Team
+# @manh-t is the Team Lead
+* @manh-t @doannimble @luongvo @markgravity
+
+# Engineering Leads
+CODEOWNERS @nimblehq/engineering-leads
```
Comment (thiennguyen0196) [inline .github/CODEOWNERS:6]: This file contains errors due to invalid user @ducbm051291 
I think we can update this quick

**Review by thiennguyen0196 — COMMENTED** [review]
Just one minor error, the rest LGTM 😄

**Review by thiennguyen0196 — DISMISSED** [review]
lgtm

**Review by thiennguyen0196 — APPROVED** [review]
LGTM

**Review by suho — APPROVED** [review]

**Pull request merged**